legal-copy/legal-templates.js

```diff
@@ -34,7 +32,7 @@ module.exports.templateEnd = `
           <a href="https://www.mozilla.org" class="mozilla-logo"></a>
           <a data-l10n-id="footerLinkLegal" href="https://www.mozilla.org/about/legal/" class="boilerplate">Legal</a>
 <a data-l10n-id="footerLinkPrivacy" href="/privacy" class="boilerplate">Privacy</a>
-<a data-l10n-id="footerLinkTerms" href="/terms" class="boilerplate">Terms of Use</a>
+<a data-l10n-id="footerLinkTerms" href="/terms" class="boilerplate">Terms</a>
```

Comment:
Actually, maybe I am as dumb as I think I am.
In an attempt to prove we're inconsistent, I've actually proven we are consistent, but our L20N is a dark abyss.

The front end [ultimately] says "Terms":

![firefox_test_pilot](https://cloud.githubusercontent.com/assets/557895/15982082/2ea5b666-2f34-11e6-9e22-2876adb7ac15.png)

The [/testpilot/frontend/static-src/app/views/footer-view.js:8](https://github.com/mozilla/testpilot/blob/637abee3e107ecf0aa587c80ef778b72c4578e41/testpilot/frontend/static-src/app/views/footer-view.js#L8) says "Terms of Use":

```html
<a data-l10n-id="footerLinkTerms" href="/terms" class="boilerplate">Terms of Use</a>
```

That casts doubt on `footLinkTerms` key in our L20N file, [/locales/en-US/app.l20n:11](https://github.com/mozilla/testpilot/blob/637abee3e107ecf0aa587c80ef778b72c4578e41/locales/en-US/app.l20n#L11), which says "Terms":

```xml
<footerLinkTerms "Terms">
```

Now I'm lost. We should probably either:
(a) change "Terms of Use" in the template to say "Terms" to avoid future confusion, or 
(b) change the L20N file (and revert my change above) so the string stays as "Terms of Use".